nupic/algorithms/KNNClassifier.py

```diff
@@ -573,38 +578,32 @@ def getDistances(self, inputPattern):
     return (dist, self._categoryList)
 
 
-  def infer(self, inputPattern, computeScores=True, overCategories=True,
-            partitionId=None):
-    """Finds the category that best matches the input pattern. Returns the
-    winning category index as well as a distribution over all categories.
-
-    @param inputPattern (list) A pattern to be classified
-
-    @param computeScores NO EFFECT
-
-    @param overCategories NO EFFECT
-
-    @param partitionId (int) UNKNOWN
-
-    This method returns a 4-tuple: (winner, inferenceResult, dist, categoryDist)
-      winner:           The category with the greatest number of nearest
-                        neighbors within the kth nearest neighbors. If the
-                        inferenceResult contains no neighbors, the value of
-                        winner is None; this applies to the case of exact
-                        matching.
-      inferenceResult:  A list of length numCategories, each entry contains the
-                        number of neighbors within the top k neighbors that
-                        are in that category.
-      dist:             A list of length numPrototypes. Each entry is the
-                        distance from the unknown to that prototype. All
-                        distances are between 0.0 and 1.0
-      categoryDist:     A list of length numCategories. Each entry is the
-                        distance from the unknown to the nearest prototype of
-                        that category. All distances are between 0 and 1.0.
+  ##########################################################################
+  def infer(self, inputPattern, numWinners=1, computeScores=True,
+                  overCategories=True, partitionId=None):
```

Comment:
Either is fine. Examples in PEP-8 use this style. But make sure to line up indentation correctly (see [PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation))